src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -50,6 +49,12 @@ object SbtSassCompiler extends AutoPlugin {
     // define how sass files should be compiled
     Assets / compileSass                   := Def
       .task {
+        val logger = streams.value.log
+
+        Try(Class.forName("org.irundaia.sbt.sass.SbtSassify"))
+          .toOption
+          .map(_ => logger.warn("The sbt-sassify plugin detected. Please remove it to avoid unexpected behaviour."))
```

Comment:
I used WARN level for this log, because I am not 100% sure if we should be using error if we know that this unexpected behaviour can happen only in some weird specific corner cases.

Using error can confuse devs that there is something wrong, but the compilation ended with success.

What do you think about it?